src/sql/src/names.rs

```diff
@@ -1033,6 +1042,9 @@ impl TryFrom<ResolvedObjectName> for ObjectId {
             ResolvedObjectName::Item(name) => match name {
                 ResolvedItemName::Item { id, .. } => Ok(ObjectId::Item(id)),
                 ResolvedItemName::Cte { .. } => Err(anyhow!("CTE does not correspond to object")),
+                ResolvedItemName::ContinualTask { .. } => {
+                    Err(anyhow!("CT does not correspond to object"))
```

Comment:
Small preference to writing out 'ContinualTask' instead of 'CT' since it might not be as recognizable as 'CTE'
```suggestion
                    Err(anyhow!("ContinualTask does not correspond to object"))
```

Reply:
Done!